orchestrator/channels/slack_adapter.py

```
@@ -0,0 +1,166 @@
+        self._server_task: Optional[asyncio.Task] = None
+
+    async def start(self):
+        """Start the Slack bot."""
+        try:
+            from slack_bolt.async_app import AsyncApp
+
+            bot_token = self.config.get("bot_token", "")
+            signing_secret = self.config.get("signing_secret", "")
+
+            if not bot_token:
+                raise ValueError("bot_token is required")
```

Ruff (0.15.0)

[warning] Abstract `raise` to an inner function

(TRY301)